codegen/templates/tchannel_server.tmpl

```diff
@@ -0,0 +1,124 @@
+{{- /* template to render edge gateway tchannel server code */ -}}
+// Code generated by zanzibar
+// @generated
+
+// Package {{.PackageName}} is generated code used to make or handle TChannel calls using Thrift.
+package {{.PackageName}}
+
+import (
+	"context"
```

Comment:
fix in https://github.com/uber/zanzibar/pull/57/commits/785e981803a15744326f7872135867214e956386